src/model.js

```diff
@@ -29,6 +29,16 @@ module.exports = function(koop) {
         let layerId = req.params.layer;
         this.esConfig = config.esConnections[esId];
         const indexConfig = this.esConfig.indices[serviceName];
+        let tileExtent = {
+            'xmin': -20037507.067161843,
+            'ymin': -20037507.067161843,
+            'xmax': 20037507.067161843,
+            'ymax': 20037507.067161843,
+            'spatialReference': {
+                'cs': 'pcs',
+                'wkid': 102100
+            }
+        };
```

Comment:
Can we make the default extent value come from the config?  Or if it isn't defined in the config then default to global. Seems like some users would not want this hard-coded.

Also maybe just call this `extent` instead of `tileExtent` since it will be used as the metadata.extent value for all routes not just tile routes?

Reply:
Updated.